### ISSUE TITLE
Reboot when no job is running

### ISSUE THREAD

**Issue by techtonik** [dshearer/jobber#232]
How ensure that no jobs are running to reboot server that needs updates?

**Comment by dshearer** [comment]
I'm sorry, I missed this.  Hm.... I guess there isn't an easy way.

**Comment by techtonik** [comment]
And if jobber was a plain translator to `systemd` commands - would this be possible?

**Comment by dshearer** [comment]
It’s possible now with the appropriate code change.

**Comment by dshearer** [comment]
No, I just mean that it would not be necessary to mess with systemd in order to add the ability to tell whether jobs are still running.

**Comment by ahathaway** [comment]
unsure if this should be part of this request or a separate one. Would like to see the ability to pause a job, and have that paused status persist reboot. I believe that is the same as this existing issue, but let me know if I need to file a new one. Thanks!
